tools/upgrade-loader-or-syncer-to-dm.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Upgrade Loader or Syncer to Data Migration
+summary: This document introduces how to upgrade Loader or Syncer to Data Migration. 
+category: tools
+---
+
+# Upgrade Loader or Syncer to Data Migration
+
+This document introduces how to upgrade Loader or Syncer to DM (Data Migration).
+
+## Upgrade Loader to Data Migration
+
+Loader is used to import the full data that is exported from [mydumper](https://github.com/pingcap/docs/blob/master/tools/mydumper.md) to TiDB.
```

Comment:
fix link

Reply:
@kennytm What's wrong with this link?

Reply:
It should point to `../tools/mydumper.md` not `github.com`?

Reply:
@kennytm Yes, sir @_@